rotkehlchen/chain/bitcoin/utils.py

```diff
@@ -146,3 +157,51 @@ def is_valid_derivation_path(path: Any) -> Tuple[bool, str]:
             return False, f'Found negative integer node {value} in xpub derivation path'
 
     return True, ''
+
+
+def scriptpubkey_to_p2pkh_address(data: bytes) -> BTCAddress:
+    """Return a P2PKH  address given a scriptpubkey
+
+    P2PKH: OP_DUP OP_HASH160 <pubKeyHash> OP_EQUALVERIFY OP_CHECKSIG
+    """
+    if (
```

Comment:
where did you get this code from?

Reply:
All the info was almost there (I attached it in the issue):
- [EIP](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2304.md#p2pkhversion).
- [ENS implementation](https://github.com/ensdomains/address-encoder/blob/master/src/index.ts#L52).

The missing piece was appending the checksum